### PR TITLE
GEOMESA-2873 Expose flow file attributes in converters

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
requires https://github.com/locationtech/geomesa/pull/2572

File: geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/ConverterIngestProcessor.scala [hunk 1]
```diff
@@ -142,7 +147,10 @@ trait ConverterIngestProcessor extends AbstractGeoIngestProcessor {
 
       val converter = converters.borrowObject()
       try {
-        val ec = converter.createEvaluationContext(EvaluationContext.inputFileParam(name))
+        val globalParams =
+          EvaluationContext.inputFileParam(name) ++
+              (if (exposeAttributes) { file.getAttributes.asScala -- Attributes.all } else { Map.empty })
```
Comment (jnh5y) [inline geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/ConverterIngestProcessor.scala:152]: Does the -- Attributes.all strip out the datastore params?  Like the password?  

If so, that's good catch.:)

**Comment by jnh5y** [comment]
Merged as ec94a64220f0ca5d70933e10b6b374ad2ab9a8f0.

File: geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/ConverterIngestProcessor.scala (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/ConverterIngestProcessor.scala:152]: no, that just strips out the geomesa.convert and geomesa.sft configs that may be present. the datastore params aren't exposed as flow file attributes, these would be other per-file metadata (like timestamps or correlation fields)

**Pull request merged**